InvenTree/stock/api.py

```diff
@@ -1074,7 +1074,7 @@ class StockItemTestResultDetail(generics.RetrieveUpdateDestroyAPIView):
     serializer_class = StockSerializers.StockItemTestResultSerializer
 
 
-class StockItemTestResultList(generics.ListCreateAPIView):
+class StockItemTestResultList(BulkDeleteMixin, generics.ListCreateAPIView):
```

Comment:
Suggestion: create a ListCreateDeleteAPIView with the same inheritance in InvenTree/InvenTree/api.py and use that as a single import everywhere we need it.